includes/wikia/services/HubService.class.php

```diff
@@ -62,26 +62,28 @@ public static function getCurrentWikiaVerticalName() {
 
 	/**
 	 * Get canonical vertical name for given cityId.
+	 * For corporate homepages (actual and hub-based) return 'fandom'.
 	 * For Lifestyle and Gaming return their names.
-	 * For corporate pages return false - we don't want to track traffic from there in Comscore
+	 * For Other return 'lifestyle'.
 	 * For rest of values return Entertainment.
+	 * 
 	 * @param $cityId
-	 * @return Boolean|String
+	 *
+	 * @return String
 	 */
 	public static function getVerticalNameForComscore( $cityId ) {
-		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $cityId );
+		global $wgDisableWAMOnHubs;
```

Comment:
That's a multidimensional question. :wink: I'm interested in any possible aspect—starting from code readability for a developer through performance to compatibility with our/MediaWiki guidelines.